README.md

```diff
@@ -94,3 +94,16 @@ $response = $gateway->createSingleUseBankToken([
 
 $singleUseTokenId = $response->getData('singleUseTokenId');
 ```
+
+## Contributing
+
+Contributions are **welcome** and will be fully **credited**.
```

Comment:
I'm not clear on what is meant by "fully credited" here. Github will keep track of contributors, but there's no current plan to create a custom list of contributors in the repo as such.

Reply:
Its like being thankful for supporting the package, if you would like to give feedback, i can update the PR. If you are not that happy with the PR, i am happy with closing it.

Reply:
@rotassator what about licensing ? Is this package following MIT license or other ?

Reply:
There's a LICENCE file for that. :)
https://github.com/rotassator/omnipay-payway-restapi/blob/master/LICENSE